components/core/src/clp_s/search/kql/CMakeLists.txt

```diff
@@ -1,25 +1,35 @@
-ANTLR_TARGET(
-        KqlParser
-        Kql.g4
-        LEXER PARSER VISITOR
-        PACKAGE kql
+set(ANTLR_GENERATED_KQL_CODE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/antlr_generated)
+
+set(ANTLR_GENERATED_KQL_CODE
```

Comment:
How about `ANTLR_GENERATED_SOURCES` (to be consistent with the variable naming in other CMake files)?